src/shared/components/event-news-meta/index.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import styles from '../events-list/style.css';
+
+import TagsList from '../tags-list';
+import EventLinksList from '../event-links-list';
+
+const EventNewsMeta = ({
+  internalLinks,
+  externalLinks,
+  tags,
+}) => {
+  const linksSection = (<div>
+    {
+      externalLinks || internalLinks ?
+        <div className={styles.eventLinks}>
+          <EventLinksList
+            linkList={externalLinks}
+            listType="external" />
+          <EventLinksList
+            linkList={internalLinks}
+            listType="internal" />
+        </div>
+      : <noscript />
```

Comment:
I find ternary statements in JSX really hard to read, would prefer if else and extracted functions where needed. Maybe it's just me though!